drv/vsc85xx/src/vsc8562.rs

```diff
@@ -97,10 +129,33 @@ impl<'a, 'b, P: PhyRw> Vsc8562Phy<'a, 'b, P> {
                 r.0 |= 25
             })?;
             v.write(phy::TR::TR_16(), 0x8fa4.into())
-        })?;
+        })
+    }
 
-        // In the SDK, there's more configuration for 100BT, which we don't use
+    // Based on `vtss_phy_sd1g_patch_private` in the SDK
+    fn sd1g_patch(&mut self, is_100fx: bool) -> Result<(), VscError> {
+        for p in 0..2 {
+            // XXX The SDK just does self.port * 2 (including any offset); based on
```

Comment:
Feels like the `XXX` isn't merited as it seems to actually be correct? ;)

Reply:
Maybe! It's hard to tell if it's actually correct, because it's not clear what this actually _does_ – I'm going to leave the comment as a breadcrumb in case we ever get suspicious of the SerDes.